tests_metricflow/snapshots/test_derived_metric_rendering.py/SqlQueryPlan/DuckDB/test_offset_to_grain_metric_filter_and_query_have_different_granularities__plan0_optimized.sql

```diff
@@ -19,8 +19,8 @@ FROM (
   FROM (
     -- Join to Time Spine Dataset
     SELECT
-      DATE_TRUNC('month', subq_12.ds) AS metric_time__month
-      , subq_10.metric_time__day AS metric_time__day
+      subq_12.ds AS metric_time__day
```

Comment:
Here you can see we are now pulling the join column from the time spine, where in the last PR we were not.